noloco/utils.py

```diff
@@ -131,25 +133,42 @@ def find_relationship_data_type(
         # If the relationship field exists on the parent data type then
         # this is a forward relationship and we can simply look up the
         # relationship data type by the corresponding field type.
-        return find_data_type_by_name(relationship_field['type'], data_types)
+        return {
+            'data_type': find_data_type_by_name(
+                relationship_field['type'], data_types),
+            'is_collection': relationship_field['relationship'] == MANY_TO_MANY
```

Comment:
Your `is_collection` calculation isn't quite right 

As you can see from `isMultiRelationship` there's always 2 scenarios that are multi 

```js
export const isMultiRelationship = (relationship) =>
  relationship === ONE_TO_MANY || relationship === MANY_TO_MANY;

export const isReverseMultiRelationship = (relationship) =>
  relationship === MANY_TO_ONE || relationship === MANY_TO_MANY;

export const isMultiField = (field) =>
  (field.relationship && isMultiRelationship(field.relationship)) ||
  (field.relatedField &&
    isReverseMultiRelationship(field.relatedField.relationship));
```

Reply:
Got it, I think that should be corrected now.